test/ui-test/testSuites/suite_messaging/tst_ChatFlow/test.feature

```diff
@@ -28,13 +28,12 @@ Feature: Status Desktop Chat Basic Flows
     @mayfail
     # TODO: It works standalone but when it runs as part of the sequence, the action of reply is not done always. The popup option does not appear.
     Scenario Outline: The user can reply to the last message
```

Comment:
Oh I thought this was how we get hold of variables. But now I realize it's not necessary

Reply:
I also use it, although it is just only one case, as a way to propagate parameters since here we have the `message` one that is used in different statements and with this we avoid hardcoded issues / mismatched params.